tests/system/apps/__init__.py

```diff
@@ -8,15 +8,15 @@ def apps_dir():
     return os.path.dirname(os.path.abspath(__file__))
 
 
-def load_app(app_def_file, app_id=None, parent_group="/"):
+def load_app(app_def_file, app_id=None, parent_group="/",  app_suffix=""):
```

Comment:
Why the suffix? Would it be enough to have it part of `app_id`?

Reply:
@jeschkies nope, because if you define an `app_id`, no additional random part will be generated. Hence the `suffix`.

Reply:
You could still use `make_app_id`. The suffix is just odd to me. It basically states includes this in my random id.

Reply:
That sounds reasonable :+1: